package.json

```diff
@@ -51,12 +50,11 @@
     "eslint-plugin-react-refresh": "^0.4.6",
     "husky": "^9.0.11",
     "lint-staged": "^15.2.2",
-    "postcss": "^8.4.38",
     "prettier": "^3.2.5",
     "prettier-plugin-tailwindcss": "^0.5.13",
     "sass": "^1.74.1",
     "tailwindcss": "^3.4.3",
     "typescript": "^5.2.2",
     "vite": "^5.2.0"
   }
-}
+}
```

Comment:
same above @quyn2904

Reply:
I use NextUI together with TailwindCSS so I need to install Tailwind CSS as a PostCSS Plugin to integrate with my build tool, Vite.